jupytercad/fcstd_ydoc.py

```diff
@@ -7,10 +7,10 @@
 class YFCStd(YBaseDoc):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self._ysource = self._ydoc.get_text('source')
-        self._yobjects = self._ydoc.get_array('objects')
-        self._yoptions = self._ydoc.get_map('options')
-        self._ymeta = self._ydoc.get_map('metadata')
+        self._ysource = self._ydoc.get_text("source")
```

Comment:
Can we keep using single quotation marks? I'm using [blue](https://pypi.org/project/blue/) as a drop-in replacement for `black` but with single quotation strings and it works pretty well.